planet/src/main/webapp/js/itinerary.js

```diff
@@ -27,6 +27,8 @@ window.addEvent = addEvent;
 window.saveEvents = saveEvents;
 window.generateItinerary = generateItinerary;
 
+const database = firebase.database();
```

Comment:
nit: Weren't you able to use the global variable in the authentication.js file?

Reply:
I wasn't. When I pulled the last PR in it I got a error and was not able to use the global variable anymore. I have no idea why since it was working fine for me until the last merge

Reply:
That's because the database global variable had to be initialized right after initializing the firebaseApp. It's okay to leave it as it is since it's in a module.